### PR TITLE
added journal messages

### PR DESCRIPTION
**Pull request by jalogisch**
After feedback from the community ( https://community.graylog.org/t/graylog-uncommited-messages-deleted-from-journal-utilization-is-too-high/3177/6 ) and some Enterprise Customers I added the error message to the FAQ and put some additional information to it.

This might be good become one chapter "troubleshooting" but currently its better to find that error in the documentation than noting. 

ppl do not read the online description of this error!

### CODE REVIEW

**Comment by jalogisch** [comment]
this need to be cherry-picked into 2.4 and 3.0 once merged

File: pages/faq.rst [hunk 1]
```diff
@@ -248,6 +248,22 @@ The best option, apart from not sending fields with dots, is to remember to writ
 
 For example, if an incoming message contains the field ``docker.container`` stream rules use that name, whereas alert conditions need to use ``docker_container``. You will notice that the search results also use the latter name.
 
+What does "Uncommited messages deleted from journal" means?
```
Comment (kroepke) [inline pages/faq.rst]: `means` -> `mean`

File: pages/faq.rst [hunk 2]
```diff
@@ -248,6 +248,22 @@ The best option, apart from not sending fields with dots, is to remember to writ
...
+-----------------------------------------------------------
+Some messages were deleted from the Graylog journal before they could be written to Elasticsearch. Please
+verify that your Elasticsearch cluster is healthy and fast enough. You may also want to review your Graylog
+journal settings and set a higher limit.
+
+Most times Graylog is not able to connect to Elasticsearch or the Elasticsearch Cluster is not able to process the ingested messages in time. Add more resources to Elasticsearch or adjust :ref:`the output settings <output_batch_size>` from Graylog to Elasticsearch.
+
+
+What does "Journal utilization is too high" means?
```
Comment (kroepke) [inline pages/faq.rst]: `means` -> `mean`

File: pages/faq.rst [hunk 3]
```diff
@@ -248,6 +248,22 @@ The best option, apart from not sending fields with dots, is to remember to writ
 
 For example, if an incoming message contains the field ``docker.container`` stream rules use that name, whereas alert conditions need to use ``docker_container``. You will notice that the search results also use the latter name.
 
+What does "Uncommited messages deleted from journal" means?
+-----------------------------------------------------------
+Some messages were deleted from the Graylog journal before they could be written to Elasticsearch. Please
+verify that your Elasticsearch cluster is healthy and fast enough. You may also want to review your Graylog
+journal settings and set a higher limit.
+
+Most times Graylog is not able to connect to Elasticsearch or the Elasticsearch Cluster is not able to process the ingested messages in time. Add more resources to Elasticsearch or adjust :ref:`the output settings <output_batch_size>` from Graylog to Elasticsearch.
```
Comment (kroepke) [inline pages/faq.rst]: `Most times` -> `This can happen when`

File: pages/faq.rst [hunk 4]
```diff
@@ -248,6 +248,22 @@ The best option, apart from not sending fields with dots, is to remember to writ
...
+
+
+What does "Journal utilization is too high" means?
+--------------------------------------------------
+Journal utilization is too high and may go over the limit soon. Please verify that your Elasticsearch cluster
+is healthy and fast enough. You may also want to review your Graylog journal settings and set a higher limit.
+
+Most times Graylog is not able to connect to Elasticsearch or the Elasticsearch Cluster is not able to process the ingested messages in time. Add more resources to Elasticsearch or adjust :ref:`the output settings <output_batch_size>` from Graylog to Elasticsearch.
```
Comment (kroepke) [inline pages/faq.rst]: `Most times` -> `This can happen when`

**Review by kroepke — CHANGES_REQUESTED** [review]

**Review by kroepke — APPROVED** [review]

**Pull request merged**